0.3.4 version is not compatible with Python versions less than 3.5

There is no information about supported versions in setup.py, so I assume 2.7 is supported.

```
    def expect(expr, msg=None):
        'keeps track of failed expectations'
        global _failed_expectations, _is_first_call
        caller = ''
    
        '''
        ensure that the call is coming from 'test*' method
        '''
        stack_list = inspect.stack()
        for stack in stack_list:
>           if stack.function.startswith('test'):
E           AttributeError: 'tuple' object has no attribute 'function'

/usr/lib/python2.7/site-packages/delayed_assert/delayed_assert.py:48: AttributeError
```

--- a/delayed_assert/delayed_assert.py
+++ b/delayed_assert/delayed_assert.py
@@ -44,8 +44,9 @@ def expect(expr, msg=None):
     '''
     stack_list = inspect.stack()
     for stack in stack_list:
-        if stack.function.startswith('test'):
-            caller = stack.function
+        func_name = getattr(stack, 'function', stack[3])
+        if func_name.startswith('test'):
+            caller = func_name
             break
 
     if caller == '':

--- a/setup.py
+++ b/setup.py
@@ -5,7 +5,7 @@ from setuptools import setup
 
 setup(
     name='delayed_assert',
-    version='0.3.4',
+    version='0.3.5',
     description='Delayed/soft assertions for python',
     long_description=io.open(os.path.join(os.path.dirname('__file__'), 'README.md'), encoding='utf-8').read(),
     long_description_content_type='text/markdown',

